### PR TITLE
Natacha user widget

### PR DESCRIPTION
**Pull request by tsipoy**
Hello Team,

I have just created the user widget. Please have a look and feel free to drop comments.

Thanks in advance.

### CODE REVIEW

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 1]
```diff
@@ -50,21 +54,60 @@ export interface LinkTypes {
...
-    <Link to={`/${path}`} data-testid={text}>
-      <img src={imgSrc} alt={alt} />
-      <span>{text}</span>
-    </Link>
-  </Item>
-)
+export const HeaderNavLink: FC<ItemType> = ({ path, text, imgSrc, alt }) => {
+  const dispatch = useAppDispatch()
+  const offers = useSelector(adsSelector)
+  console.log(offers)
```
Comment (ganamavo) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:60]: We don't want this console to be merged into the base branch

File: src/components/Widget/Widget.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import { Paths } from '../../paths'
+
+export interface WidgetsProps {
+  src?: string;
+  alt?: string;
+  offersNumber?: number;
+  logOutLabel?: string;
+  offersLabel?: string;
+  profileName?: string;
+}
```
Comment (ganamavo) [inline src/components/Widget/Widget.tsx:14]: By making all props optional, the component can be used without these data so none of these props is required

**Comment by ganamavo** [comment]
Is this done @tsipoy

**Comment by andrzej-onja** [comment]
closing as it was a part of graded project only

**Pull request closed without merging**